### PR TITLE
feat: put toggle + disclaimer

### LINKED ISSUE #234: Create a V2/V1 toggle in QueryAPI.App

### PR DESCRIPTION
**Pull request by roshaans**
Adds a toggle to update which graphql endpoint the logs and state are pulled from. 

 
gipp

### CODE REVIEW

File: frontend/widgets/src/QueryApi.IndexerStatus.jsx [hunk 1]
```diff
@@ -244,6 +291,39 @@ return (
...
+            justifyContent: "center",
+          }}
+        >
+          <DisclaimerContainer>
+            <div className="flex">
+              <Notice>V2 Testing Notice</Notice>
+              <div style={{ display: "flex" }}>
+                <DisclaimerText>
+                  QueryAPI is still in beta. We are working on a OueryAPI V2
+                  with faster historical processing, easier access to DB and and
```
Comment (darunrs) [inline frontend/widgets/src/QueryApi.IndexerStatus.jsx:308]: nit: two "and"s next to each other.

File: frontend/replacement.dev.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
     "REPL_ACCOUNT_ID": "dev-queryapi.dataplatform.near",
     "REPL_GRAPHQL_ENDPOINT": "https://near-queryapi.dev.api.pagoda.co",
+    "REPL_GRAPHQL_ENDPOINT_V2": "https://queryapi-hasura-graphql-mainnet-vcqilefdcq-ew.a.run.app",
```
Comment (darunrs) [inline frontend/replacement.dev.json:4]: I see the V1 and V2 endpoint don't look similar. Is the V1 endpoint a domain we registered or something?

File: frontend/widgets/src/QueryApi.IndexerStatus.jsx [hunk 3]
```diff
@@ -105,11 +109,15 @@ State.init({
   indexer_resPage: 0,
   logsPage: 0,
   statePage: 0,
+  v2Toggle: false,
```
Comment (darunrs) [inline frontend/widgets/src/QueryApi.IndexerStatus.jsx:112]: Could we pull the v2Toggle from storage to retain the user's previous selection, and default to false if none? If someone has to refresh their page to view logs for example, and it switches to off each time, that could be frustrating. 

Also, does the v2Toggle remain when switching indexers or is it switched back to off when you switch indexers?

File: frontend/widgets/src/components/toggle.jsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    {...rest}
+  >
+    <ToggleLabel htmlFor={`toggle-${key}`}>{label}</ToggleLabel>
+
+    <ToggleSwitchRoot
+      checked={active}
+      className="shadow-none"
+      id={`toggle-${key}`}
+      onCheckedChange={disabled ? null : onSwitch}
+      title={disabled ? `Permanently ${active ? "enabled" : "disabled"}` : null}
```
Comment (darunrs) [inline frontend/widgets/src/components/toggle.jsx:70]: What does this line do? What is supposed to be "permanently enabled/disabled"?

Comment (roshaans) [reply frontend/widgets/src/components/toggle.jsx:70]: Not quite sure actually. I copied this component from devgogs.near

File: frontend/widgets/src/QueryApi.IndexerStatus.jsx (hunk 3 above)
Comment (roshaans) [reply frontend/widgets/src/QueryApi.IndexerStatus.jsx]: Good point. I tried to implement it but it was a little finicky.

**Pull request merged**